R/pkg/inst/worker/worker.R

```diff
@@ -27,6 +27,54 @@ elapsedSecs <- function() {
   proc.time()[3]
 }
 
+compute <- function(mode, partition, serializer, deserializer, key,
```

Comment:
function(computeFunc, mode, partition, key, inputData, serializer, deserializer, colNames) 
remove unused argument and re-order arguments